charts/stable/omada-controller/templates/common.yaml

```diff
@@ -1,24 +1,9 @@
 {{/* Render the templates */}}
-{{ include "tc.v1.common.loader.init" . }}
 
 {{/*
-Omada automatically redirects to that port.
-Instead of consuming the external 443 port, it's better to switch internal
-But still give user the ability to change it.
-When ingress is enabled, set this to 443, otherwise set it to whatever `port` is set to
+The old behavior of always running on port 443 seams to have changed upstream.  The 
+current upstream behavior is to not allow the port to be changed (without re-install) 
```

Comment:
It was always only on first install I think.
But I remember it took a while to figure that this was needed. Otherwise when ingress was enabled it wouldn't function correctly with the devices.

@StevenMcElligott Do you remember maybe better?

Reply:
You're right, basically if we don't put this script even if it's 8043 on initial install we get Gateway errors and problems with ingress. If it changed upstream which I'm not aware of then it makes sense and we can try

Reply:
Yea... the current release doesn't behave (specifically ONLY firmware update is broken) UNLESS I create a custom ingress pointing to the internal 8043 port (and disable the DEFAULT ingress).  Currently this doesn't work because it expects the at least one ingress to be labeled primary (so I can't disable primary ingress). I'll look more into it and try to pinpoint the upstream change.